Neos.Flow/Classes/ResourceManagement/ResourceRepository.php

```diff
@@ -74,6 +74,9 @@ public function __construct()
      */
     public function add($object)
     {
+        if ($object instanceof PersistentResource && $object->isDeferred()) {
```

Comment:
@kitsunet see my other comment - I think this is a good measure to prevent the primary line of accidential misuse